include/mbgl/util/projection.hpp

```diff
@@ -78,8 +78,9 @@ class Projection {
         return project_(latLng, worldSize(scale));
     }
 
-    static Point<double> project(const LatLng& latLng, uint8_t zoom) {
-        return project_(latLng, std::pow(2.0, zoom));
+    //Returns point on tile
+    static Point<double> project(const LatLng& latLng, int32_t zoom) {
```

Comment:
This method projects to a tile fraction and not to a tile-size based world. I'm not sure if it should be called `project` or exposed from `TileCoordinate`